controllers/users.js

```diff
@@ -195,6 +195,37 @@ const getUsernameAvailabilty = async (req, res) => {
   }
 };
 
+const getUsername = async (req, res) => {
+  try {
+    const { firstname, lastname, dev } = req.query;
+    if (dev) {
+      const baseUsername = `${firstname.toLowerCase()}-${lastname.toLowerCase()}`;
+      let newUsername = baseUsername;
+      let count = 1;
+
+      while (true) {
```

Comment:
is there no other way then `while(true)` ?

Reply:
Done with changes @apurvkhare...can you please check latest commit

Reply:
But @apurvkhare I am going to change my approach because in this approach I am accessing the database every time to check for the existence of a username but accessing database every time is not cost effective 
So my new approach is  access database once and fetch all user that matches the given firstname and lastname.

Reply:
What I see now is `for` with no end condition which is the same, are you working on updating this or that's the final commit?